### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-ansible-role.git`](https://github.com/cisagov/skeleton-ansible-role.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/ansible-role-mongo.git ansible-role-mongo
    cd ansible-role-mongo
    git remote add skeleton https://github.com/cisagov/skeleton-ansible-role.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add README.md 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: molecule/default/python.yml [hunk 1]
```diff
@@ -1,11 +1,15 @@
 ---
 - hosts: all
-  name: Install python2/python3 and pip2/pip3
+  name: Install python2 and pip2
```
Comment (jsf9k) [inline molecule/default/python.yml]: Why did you change this?  With the given `vars` the roles included below will in fact install both versions.

Comment (jsf9k) [reply molecule/default/python.yml]: Also, if Python is a real dependency of this role then it should be listed as such in `meta/main.yml` as well as in `README.md`, and you should let the installation take place that way instead of via the `prepare.yml` playbook.

**Review by jsf9k — CHANGES_REQUESTED** [review]

File: molecule/default/python.yml (hunk 1 above)
Comment (dav3r) [reply molecule/default/python.yml]: @jsf9k I think I did things correctly in https://github.com/cisagov/ansible-role-mongo/pull/12/commits/c4d66bed20078a745cb286ac282a57abf25a1cc9, but please double-check to make sure.

Comment (jsf9k) [reply molecule/default/python.yml]: I think this is correct now.

**Review by jsf9k — APPROVED** [review]

File: meta/main.yml [hunk 2]
```diff
@@ -52,3 +54,5 @@ dependencies:
     name: numactl
   - src: https://github.com/cisagov/ansible-role-pip
```
Comment (mcdonnnj) [inline meta/main.yml:55]: Please add [cisagov/ansible-role-python](https://github.com/cisagov/ansible-role-python) as a dependency since we are installing Python packages with `pip`.

Comment (dav3r) [reply meta/main.yml:55]: 👍 Please see https://github.com/cisagov/ansible-role-mongo/pull/12/commits/71ed25bd6a9e432678af68a5d760eb09d1e7dd73.

**Review by mcdonnnj — APPROVED** [review]
